installer/scripts/service_control

```diff
@@ -93,6 +102,7 @@ start_omsagent_process()
 
 start_omsagent()
 {
+    exit_if_agent_not_onboarded
```

Comment:
Because of prior implementation, you can be in a situation where we are onboarded with no service registered (although that was never released to customers). Did you consider just registering the service prior to a "start" or "restart" request if we are onboarded but not registered?